Fix broken gopxl/beep/v2 pseudo-version in go.mod

Upstream references commit defe79638e99 which no longer exists in the
gopxl/beep repository. Use the actual release tag v2.1.1 instead and
remove the unnecessary replace directive.

--- a/go.mod
+++ b/go.mod
@@ -2,7 +2,6 @@
 
 go 1.20
 
-replace github.com/gopxl/beep/v2 => github.com/gopxl/beep/v2 v2.1.1-0.20240921133731-defe79638e99
 
 require (
 	github.com/assemblaj/ggpo v0.0.0-20241121212120-467f3782a9cf
@@ -10,7 +9,7 @@
 	github.com/go-gl/gl v0.0.0-20231021071112-07e5d0ea2e71
 	github.com/go-gl/glfw/v3.3/glfw v0.0.0-20240506104042-037f3cc74f2a
 	github.com/go-gl/mathgl v1.0.0
-	github.com/gopxl/beep/v2 v2.1.1-0.20240921133731-defe79638e99
+	github.com/gopxl/beep/v2 v2.1.1
 	github.com/ikemen-engine/reisen v0.1.10-0.20250928163542-0bb3c3392852
 	github.com/lukegb/dds v0.0.0-20190402175749-8b7170e64003
 	github.com/qmuntal/gltf v0.24.2
